media/com_associations/css/sidebyside.css

```diff
@@ -9,9 +9,6 @@
 }
 
 .sidebyside #left-panel .inner-panel {
-	border-style: solid;
-	border-color: #999999;
-	border-width: 0 1px 0 0;
 	padding: 0 10px 0 0;
```

Comment:
if you remove the border css styles here you should remove the border in `html[dir=rtl] .sidebyside #left-panel .inner-panel` too...